### ISSUE TITLE
Move definition contrat to Spec Editors Community Group

### ISSUE THREAD

**Issue by dontcallmedom** [tabatkins/bikeshed#2088]
The [bikeshed documentation](https://tabatkins.github.io/bikeshed/#dfn-contract) is at the moment the authoritative place that specifies how definitions should be marked up.

This specification has now at least two consumer implementations (shepherd and reffy), and two or three producers (bikeshed, respec and at least some support from wattsi) and as part of our work on browser-specs and reffy, we have already identified possible clarifications needed:
  https://github.com/w3c/reffy/issues/336
and are interested in approaching other editing tools from other communities (TC39, IETF, Khronos) to gauge their interest in adopting it, which would probably involve expanding the taxonomy of definitions.

Given the recent launch of the Spec Editors Community Group, this would feel like a good host to maintain this definition contract as a document of its own. I'm happy to seed such a document if moving it away from the bikeshed documentation is acceptable.

cc @tidoust @marcoscaceres

**Comment by tabatkins** [comment]
Yup, this sounds good to me. Is the SECG a GH org (in which case I can spin the contract out into a repo) or a repo under w3c (in which case I just need edit rights)?

**Comment by dontcallmedom** [comment]
I've created https://github.com/speced/spec-dfn-contract/  - feel free to overwrite its history

**Comment by marcoscaceres** [comment]
@tabatkins, I made a start at moving the contract over:
https://github.com/speced/spec-dfn-contract/pull/10

Would be great to collaborate on it.
